app/views/partials/_btn-follow.html.haml

```diff
@@ -0,0 +1,3 @@
+- following = current_user.following?(user).to_s
```

Comment:
Let's avoid creating variables in views as much as possible. Let's send this as a local variable when we call the partial